docs/static/getting-started-with-logstash.asciidoc

```diff
@@ -144,6 +144,12 @@ WARNING: The repositories do not work with older rpm based distributions
 
 See the <<running-logstash,Running Logstash>> document for managing Logstash as a system service.
 
+=== Docker
+
+An image is available for running Logstash as a Docker container. It is
+available from the Elastic Docker registry. See the <<docker>> section for
+details on how to configure and run Logstash Docker containers.
```

Comment:
Same comment as before: Would say, `See <<docker>> for
details about how to configure and run Logstash Docker containers.`